README.md

```diff
@@ -79,6 +80,7 @@ The creators of the below listed watchfaces are happy to answer your questions a
 
 | Watchface - File | License |
 | --- | --- |
+| analog-neon-diver - [Bebas Neue](analog-neon-diver/usr/share/fonts/BebasNeueBold.ttf) | The font "Bebas Neue" are licensed under SIL Open Font License and was created by [Dharma Type](http://www.dharmatype.com/). [license](http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL) |
```

Comment:
What about the leaguespartan font ?

Reply:
We have League Spartan in the "masked-spartan" stock watchface already.
I figured it might be allowed to be lazy due to the license already being cleared for the stock installation?
Anyway, happy to correct that and add it to the README.md.
Same btw for Raleway font. That one i listed in the README.md license section but it had been in stock all along.
Whats the rule for the future? List all used fonts in the unofficial-watchface README.md even tho they are delivered in stock?

Reply:
Ah sorry missed that, np then